services/frontend-service/package.json

```diff
@@ -1,95 +1,95 @@
 {
-  "name": "raw-react-app",
```

Comment:
can you do this change without reformatting?
I feel like this will revert when someone else installs/removes a package.